lib/ansible/plugins/lookup/dict.py

```diff
@@ -10,12 +10,20 @@
     short_description: returns key/value pair items from dictionaries
     description:
         - Takes dictionaries as input and returns a list with each item in the list being a dictionary with 'key' and 'value' as
-          keys to the previous dictionary's structure.
+          keys to the previous dictionary's structure. The new keys 'key' and 'value' can be overridden using the optional parameters key_name and value_name.
     options:
         _terms:
             description:
                 - A list of dictionaries
             required: True
+        key_name:
+            description:
+                - Use it to override the 'key' key name in the dictionaries. By default it will be set to 'key'.
+            required: False
+        value_name:
+            description:
+                - Use it to override the 'key' key name in the dictionaries. By default it will be set to 'value'.
+            required: False
```

Comment:
missing, `version_added` and `default`, which also means you don't need to state it in the description.

Reply:
I added `default`, so get_options defaults to `key` and `value`. Do I need to add `version_added` as well? Or would you prefer that I remove `description`?
Thank you!